templates/certificates/dynamic-styles.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Single certificate dynamic styles.
+ *
+ * Output in the header, via the `wp_head` action.
+ *
+ * @package LifterLMS/Templates/Certificates
+ *
+ * @since [version]
+ * @version [version]
+ *
+ * @param LLMS_User_Certificate $certificate      Certificate object.
+ * @param string                $width            Width (with unit) accounting for the orientation value.
+ * @param string                $height           Height (with unit) accounting for the orientation value.
+ * @param string                $background_color Background color value.
+ * @param string                $background_img   Image source URL for the background image.
+ * @param string                $padding          Internal margin value with units, ready to be used in CSS.
+ */
+
+defined( 'ABSPATH' ) || exit;
+?>
+<style type="text/css">
+	html, body {
+		background-color: <?php echo $background_color; ?>;
+	}
+	.llms-certificate-container {
+		background-image: <?php echo "url( {$background_img} )"; ?>;
+		height: <?php echo $height; ?>;
```

Comment:
```suggestion
		background-image: <?php echo "url( {$background_img} )"; ?>;
		background-size: cover;
		height: <?php echo $height; ?>;
```

Reply:
I messed up and checked out the SCSS changes I made with this PR... I've updated. `cover` doesn't seem appropriate to me in this scenario. We are defining explicit widths for the users and they can choose to upload something that is expected to be that size. Just a bit of testing with `cover` results in default certificates being cut and not displayed well so I think using `100% 100%` (and stretching it a bit) is preferable in this case.